### ISSUE TITLE
Reach compiler failure cites ai.lum class

### ISSUE THREAD

**Issue by hickst** [clulab/reach#487]
I'm trying to update Reach to use the newly released Bioresources 1.1.22 and Processors 6.0.3 but the updated build file won't compile the project. When Processors 6.0.3 is specified for Reach, it fails to compile, citing missing dependencies and mentioning `ConfigUtils.class` and `com.typesafe.config` (see compiler errors below).

Both Reach and Processors use version 1.2.1 of `com.typesafe.config` and version 0.0.7 of `ai.lum.common`. AI Lum 0.0.7  uses the newer 1.3.0 version of `com.typesafe.config`.

I tried several experiments, totally removing my Ivy cache before each:
1. I then tried updating Reach `main/build.sbt` to 1.3.0, but this also failed to compile 
2. Reach `assembly/build.sbt` file was recently updated to use `ai.lum.common 0.0.7` so I reverted `assembly/build.sbt` to use `ai.lum.common 0.0.5` but that also failed to compile.
3. Using `processors 6.0.2` also allows Reach to compile and all tests pass.

I've checked the problematic build file into a separate branch: `lum-clash`

```[info] Compiling 35 Scala sources to /Users/hickst/workspaceCLU/clulab/reach/assembly/target/scala-2.11/classes...
[info] Compiling 13 Scala sources to /Users/hickst/workspaceCLU/clulab/reach/export/target/scala-2.11/classes...
[info] Compiling 3 Scala sources to /Users/hickst/workspaceCLU/clulab/reach/target/scala-2.11/classes...
[error] missing or invalid dependency detected while loading class file 'ConfigUtils.class'.
[error] Could not access type ConfigMemorySize in package com.typesafe.config,
[error] because it (or its dependencies) are missing. Check your build definition for
[error] missing or conflicting dependencies. (Re-run with `-Ylog-classpath` to see the problematic classpath.)
[error] A full rebuild may help if 'ConfigUtils.class' was compiled against an incompatible version of com.typesafe.config.
[error] one error found
[error] (root/compile:compileIncremental) Compilation failed
```

**Comment by hickst** [comment]
Some additional, possibly related, info: Reach is also using `ai.lum.nxmlreader 0.0.7` which, in turn uses, a very old version of `ai.lum.common`: **version 0.0.1!**
Thus, both 0.0.1 and 0.0.7 are being downloaded into the Ivy cache so perhaps the compiler is choking on this.

**Comment by hickst** [comment]
Another datapoint:
5. I compiled a locally published Processors (6.0.4-SNAPSHOT) and then wiped the `.ivy/cache` completely and recompiled Reach against the snapshot: it worked! I'm very confused...perhaps the published 6.0.3 version of Processors is bad somehow?

Anyway, I would still suggest that we update all ai.lum libraries to use consistent dependencies and recompile against the updated ai.lum libraries.

**Comment by hickst** [comment]
As we discussed, I will clone the ai.lum.nxmlreader project, update the dependency, test it and report back. If this fails to solve the problem we may try to re-release Processors.

**Comment by myedibleenso** [comment]
Why would processors need to be re-released?  the nxml reader isn't a dependency of processors, right?

**Comment by hickst** [comment]
Right, but if the update of nxmlreader doesn't solve the problem, then I have to assume that the Processors release is damaged somehow. Especially since 6.0.2 and 6.0.4-snapshot do not exhibit the problem.

**Comment by myedibleenso** [comment]
Ah, got it.

**Comment by hickst** [comment]
This is closed by pull request #489
